builtin/providers/azure/provider.go

```diff
@@ -0,0 +1,43 @@
+package azure
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+// Provider returns a terraform.ResourceProvider.
+func Provider() terraform.ResourceProvider {
+	return &schema.Provider{
+		Schema: map[string]*schema.Schema{
+			"settings_file": &schema.Schema{
```

Comment:
Would have been nice to provide the id + certificate + url authentication option as well for people with Windows Azure Packs.